examples/basics/lists/app-object-list.js

```diff
@@ -0,0 +1,42 @@
+const createSession = require('../../session');
+
+const properties = {
+  qInfo: {
+    qType: 'my-list',
+  },
+  qAppObjectListDef: {
+    qType: 'my-object',
+    qData: {
+      title: '/meta/title',
+    },
+  },
+};
+
+const session = createSession();
+
+// Open the session and create a session document:
+session
+  .open()
+  .then(global => global.createSessionApp())
+  .then((doc) => {
+    // Create 10 objects of type my-object with unique titles
+    for (let i = 0; i < 10; i += 1) {
+      doc.createObject({
```

Comment:
Add each object creation promise to an array and wait for all of them to complete before creating the list object.